.github/workflows/js-css-coding-standards.yml

```diff
@@ -0,0 +1,43 @@
+name: JS and CSS Coding Standards
+
+on:
+  push:
+    branches: [ trunk, develop ]
+  pull_request:
+    branches: [ trunk, develop ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Adding Node.js setup to run JS and CSS linters
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '18.15'  # project version
```

Comment:
Do you think we could automate this number by replacing `node-version` with `node-version-file: 'package.json'`? https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file